packages/widget-v2/src/hooks/useCreateCosmosWallets.tsx

```diff
@@ -90,12 +90,13 @@ export const useCreateCosmosWallets = () => {
         return address;
       };
       const walletInfo = getCosmosWalletInfo(wallet);
```

Comment:
`getCosmosWalletInfo` is from graz, ideally we should update the response type to be " | undefined" since sometimes it can return undefined